### DISCUSSION TITLE
Custom wall thickness

### DISCUSSION THREAD

**Discussion by mikeneron** [kennetek/gridfinity-rebuilt-openscad#17] [Ideas]
First of all, thank you for putting the openscad files out there. I had designed my own parts boxes/grids and while I may keep going with that, now that I have found Gridfinity, there may not be a reason to continue. The only issue I am having with the OpenSCAD method are the thin walls. I am using a 0.8 mm nozzle to print with so it's only doing 1 wall (1 mm line width). In a couple places where the fillet is, there are actually holes in the wall. It's also easy to break the walls since there is only 1. It would be nice to be able to set a custom thickness. In my case set it to 2 mm so it prints 2 walls.

**Comment by kennetek** [comment]
At first, I was going to tell you to edit the value `d_wall` inside the file `gridfinity-constants.scad`. That's the value that is referenced everywhere in the script. However, it runs into trouble when the value is increased past 1.15. I'll look into fixing that.

**Comment by kennetek** [comment]
Squashing that bug was a little annoying, there was some CGAL error I've never seen before, and it refused to elaborate. Adding some random 0.001 shifts to a specific cutter made it work though, somehow, so you can have your 2mm thick walls now.

**Reply by mikeneron** [reply]
Thank you very much! I was able to adjust the d_wall value and it worked!

**Reply by rhaynie59** [reply]
Is this fix in the latest version in github?  I can't make the wall width 1.5

**Comment by ablackbu** [comment]
Came to say thanks for this fix.
